lambdaclass/madara_explorer
# Net drop down

## Changes:
- Add a dropdown to select the network that is displayed.
- RPC for each network.
- Application env variables for each network.
- Cache for each network.

## How to test:
- This PR adds new env variables for each testnet network:
   1. "TESTNET_RPC_API_HOST"
   2. "TESTNET_2_RPC_API_HOST"
   You'll need to set them before trying this.
- Once you have said variables set, start the application and you should be able to use the dropdown.
